project/AkkaBuild.scala

```diff
@@ -41,6 +41,7 @@ object AkkaBuild extends Build {
   val requestedScalaVersion = System.getProperty("akka.scalaVersion", "2.10.5")
   val Seq(scalaEpoch, scalaMajor) = """(\d+)\.(\d+)\..*""".r.unapplySeq(requestedScalaVersion).get.map(_.toInt)
   val streamAndHttpVersion = "2.0.1"
+  val streamAndHttpBinCompVersion = "2.0"
```

Comment:
Probably does not matter, but shouldn't 2.0.1 be the baseline (given that 2.0 was flawed)?